### DISCUSSION TITLE
Unexpected switch to maintenance mode

### DISCUSSION THREAD

**Discussion by StepanKukharskiy** [pockethost/pockethost#280] [Q&A] [answered]
Hey there! Twice in the past few days, my instance unexpectedly switched to maintenance mode. I'm curious to know if this was due to my instance or if there were any major updates happening on PocketHost. Here is the logs screenshot when it happens. Thanks!

<img width="1440" alt="IMG_1394" src="https://github.com/benallfree/pockethost/assets/47927208/c2d83efc-5a97-4096-b114-864271f5c5b8">

**Comment by benallfree** [comment]
@ganigeorgiev Do you recognize the `concurrent map writes` error? PocketHost is configured to set the instance to maintenance mode if there is any nonzero exit code.

**Reply by ganigeorgiev** [reply, marked as answer]
I think this was fixed in [v0.14.4](https://github.com/pocketbase/pocketbase/releases/tag/v0.14.4).

**Reply by benallfree** [reply]
@StepanKukharskiy What version are you running?

**Reply by StepanKukharskiy** [reply]
0.11.0

**Reply by StepanKukharskiy** [reply]
I’ll try updating then. Thanks!

**Reply by benallfree** [reply]
Great @StepanKukharskiy please let us know if that fixes the issue. It will help others who might have similar problems.

**Reply by StepanKukharskiy** [reply]
I think everything is fine now after the update. Thanks for helping out!
